akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -167,6 +185,7 @@ abstract class ActorRef extends java.lang.Comparable[ActorRef] with Serializable
  * There are implicit conversions in package.scala
  * from ActorRef -&gt; ScalaActorRef and back
  */
+@deprecated("tell method is now provided by ActorRef trait", "2.6.13")
 trait ScalaActorRef { ref: ActorRef with InternalActorRef with ActorRefScope =>
```

Comment:
Is getting this deprecation be possible to avoid in Scala 2.12 and 2.13 given that there is no explicit import to remove?

I wonder if we shouldn't just skip the deprecation, since it is source compatible we can probably anyway remove the implicit conversion for next minor Akka (where we don't promise bin comp)

Reply:
I added the deprecation just in case. But I agree, it is highly unlikely that someone is directly using this, if preferred I can remove it.

Reply:
Ah, I missed a step in my thought process, it's not emitted when it is used by the implicit conversion, only if you'd be extending this, and even though it isn't marked as `@DoNotInherit` / `Not for user extension` it is in practice.

Fine to leave the deprecation here then.